docs/api-docs/storefront/graphql/graphql-product-reviews.mdx

```diff
@@ -0,0 +1,191 @@
+# Product Reviews with the GraphQL Storefront API
+
+BigCommerce's GraphQL Storefront API allows you to create and query [product reviews](https://support.bigcommerce.com/s/article/Managing-Reviews) through the built-in comment system. 
+
+
+## Authentication
+
+To work with product reviews, the [API account](/docs/start/authentication/api-accounts) that generates [Storefront tokens](/docs/rest-authentication/tokens) must include one of the following [OAuth scopes](/docs/start/authentication/api-accounts#oauth-scopes):
+
+| UI Name | Permission | Parameter |
+|:--------|:-----------|:----------|
+| Content | modify | `store_v2_content` |
+| Content | read-only | `store_v2_content_read_only` |
+
+## Store settings
+
+There are a few store settings to consider when creating a product review:
+
+- [Overall enable product reviews](https://support.bigcommerce.com/s/article/Managing-Reviews?language=en_US#enable)
+- [Only accept product reviews from past customers](https://support.bigcommerce.com/s/article/Managing-Reviews?language=en_US#enable)
+- [Enable reCAPTCHA](https://support.bigcommerce.com/s/article/Updating-Themes-with-reCAPTCHA-v2)
+
+**Overall enable product reviews**
+
+The merchant must enable this overall setting for the review submission process to work. Disabling this setting cancels any attempts to submit reviews.
+
+**Only accept product reviews from past customers**
+
+When the merchant enables the 'Only accept product reviews from past customers' setting, only logged-in customers (not guest shoppers) can submit a review, and they must have at least one previous order placed in the store.
+This setting ensures the merchant has a genuine customer to contact if needed. You can use the [login mutation](/docs/storefront/graphql#customer-login) to log in a customer.
+Once a purchase has been made, customers can review any item in the store. The order doesn't have to include the product being reviewed. 
+
+When the merchant disables the setting, guest shoppers can also submit reviews, but they must provide an email. Since guests are not logged in, they must supply the email in the request as an input parameter.
```

Comment:
```suggestion
When the merchant turns off the setting, guest shoppers can also submit reviews but they must provide an email. Since guests do not log in, they must supply the email in the request as an input parameter.
```